Revamp query expression related BBEs (#3901)
ballerina-platform/ballerina-distribution: examples/iterating-over-xml-with-query/iterating_over_xml_with_query.md

# Iterating over XML with query

> An XML value can be iterated by using the query expression. Query expression will iterate for each sequence item.

Reviewer: ```suggestion
An XML value can be iterated by using a query expression. A query expression will iterate over each sequence item.
```

::: code creating_tables_with_query.bal :::

::: out creating_tables_with_query.out :::

## Related links

> - [Query expressions - Ballerina by example](https://ballerina.io/learn/by-example/query-expressions)

Reviewer: ```suggestion
- [Query expressions](https://ballerina.io/learn/by-example/query-expressions)
```
Fix all occurrences.
